walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,100 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    // duration is longer than an entire day
+    if(request.getDuration() >= TimeRange.END_OF_DAY)
+        return new ArrayList<TimeRange>();
```

Comment:
Nit: I have found `if`s without curly braces `{}` to be kind of dangerous since it's easy to add another indented line and miss that only the next line will be evaluated. Up to you though!